docs/migration.md

```diff
@@ -851,6 +854,7 @@ Below is an example of an OpenTelemetry configuration for a Syslog Source.
 extensions:
   sumologic:
     installation_token: <installation_token>
+    api_base_url: <api_base_url>
```

Comment:
We don't need it for customers. Default `api_base_url` works fine for all production deployments